pkgs/test_core/lib/src/util/io.dart

```diff
@@ -54,9 +53,18 @@ SuitePlatform currentPlatform(Runtime runtime) => SuitePlatform(runtime,
 ///
 /// Also returns an empty stream for Fuchsia since Fuchsia components can't
 /// access stdin.
-final stdinLines = StreamQueue(
+StreamQueue<String> get stdinLines => _stdinLines ??= StreamQueue(
     Platform.isFuchsia ? Stream<String>.empty() : lineSplitter.bind(stdin));
 
+StreamQueue<String>? _stdinLines;
+
+/// Call cancel on [stdinLines], but only if it's been accessed previously.
+void cancelStdinLines() {
+  if (_stdinLines != null) {
+    _stdinLines!.cancel(immediate: true);
+  }
```

Comment:
```suggestion
  _stdinLines?.cancel(immediate: true);
```

Sorry, forgot that we can do multiline suggestions now.

Remove the conditional and only use `?.`.